Improve HA install steps (#960)
rancher/docs: content/rancher/v2.x/en/installation/ha/helm-rancher/tls-secrets/_index.md

### Private CA Signed - Additional Steps

> If you are using a private CA, Rancher will need to have a copy of the CA certificate which is used by the Rancher Agent to validate the connection to the server.

Reviewer: ```suggestion
If you are using a private CA, Rancher requires a copy of the CA certificate, which Rancher Agent uses to validate the connection to the server.
```
